TheaterDatabase/dbo/Tables/Account.sql

```diff
@@ -0,0 +1,20 @@
+﻿CREATE TABLE [dbo].[Account] (
+    [AccountId] INT           NOT NULL,
+    [Password]  NVARCHAR (60) NOT NULL,
```

Comment:
for beginning we will implement storing settings using unique device id, so make password, username, firstname etc. optional
and add varchar property for storing identifier